groups.json

```
@@ -3,6 +3,10 @@
     "display": "Bitmap",
     "description": "Operations on the Bitmap data type"
   },
+  "bloom": {
+    "display": "Bloom",
```

Use the full name? Compare to "Sorted Set".

```suggestion
    "display": "Bloom Filter",
```

---

Updated to your suggestion